repository/config/ca.pan

```diff
@@ -18,10 +18,13 @@ default = YUM_SNAPSHOT_DATE
 required = no
 }
 variable YUM_CA_RPM_SNAPSHOT_DATE ?= if ( is_null(YUM_CA_RPM_SNAPSHOT_DATE) ) {
```

Comment:
I don't follow this code. If YUM_CA_RPM_SNAPSHOT_DATE is not null or undef then this code will not run because of the ?= But if it is_null then you return SELF which will set it to undef. The addition of debug() statements seem to be a plaster over a block that should just be re-written to be less convoluted?